tests/karma.conf.base.js

```diff
@@ -17,7 +17,8 @@ const files = [
     included: false,
   },
   {
-    pattern: "node_modules/@(idb|hyperhtml|marked|webidl2)/**/*.js",
+    pattern:
+      "node_modules/@(idb|hyperhtml|marked|webidl2|sniffy-mimetype)/**/*.js",
```

Comment:
Nit: use backticks to make prettier move it into same line as the key?

Reply:
I imagine prettier will eventually fix this.